### DISCUSSION TITLE
设计风格改动：更圆润的 MenuItem

### DISCUSSION THREAD

**Discussion by dancerphil** [ant-design/ant-design#37490] [Ideas]
在我们内部视觉评审的时候，我们的设计师给出了一个比较新颖的设计（在 antd@4 看来）

![image](https://user-images.githubusercontent.com/7264444/189260699-cdca8a3d-98ab-4ac6-b63f-332bd511d1dc.png)

我感觉这个设计观念，应该是和 antd@5 的设计方向吻合的。（从最近的 Segment、Table 等一系列改动来看）

为了不至于 antd@5 发出后，与我们想象的太不一样，所以这边把这个想法提出来，看看 antd 的设计同学是怎么打算的。

**Comment by MadCcc** [comment]
确实打算这样做的，马上就会更新上去

**Comment by afc163** [comment]
巧了。
